_docs/aws_resources.md

```diff
@@ -8,6 +8,168 @@ permalink: /aws_resources.html
 
 # AWS Resources, SDK Links, Tutorials, and Helpful Tidbits
 
+## SDK Links
+
+- [JavaScript](https://aws.amazon.com/sdk-for-browser/)
+
+- [Python](https://aws.amazon.com/sdk-for-python/)
+
+- [PHP](https://aws.amazon.com/sdk-for-php/)
+
+- [.NET](https://aws.amazon.com/sdk-for-net/)
+
+- [Ruby](https://aws.amazon.com/sdk-for-ruby/)
+
+- [Java](https://aws.amazon.com/sdk-for-java/)
+
+- [Go](https://aws.amazon.com/sdk-for-go/)
+
+- [Node.js](https://aws.amazon.com/sdk-for-node-js/)
+
+- [C++](https://aws.amazon.com/sdk-for-cpp/)
+
+### Still looking for more? [Here's some additional tools!](https://aws.amazon.com/getting-started/tools-sdks/)
+
+## Importing Large Datasets
+
+When considering importing and exporting database information, there are a number of approaches a developer can take. Two common approaches involving creating your own function, using Lambda, or any other scripting service, or using a pipeline. This section covers some of the different use cases for those options in AWS, as well as provides some sample code for exporting/importing by hand.
+
+### Pipeline
+When considering how to import data sets, AWS's Data Pipeline service is a service that eleminates many of the manual steps a developer would have to take. It is a process that is flexible to the dataset, scalable, and easily processed in parallel - which means it can process a lot of data very quickly.
+
+As a turnkey solution, Data Pipeline works with most datasets, and can be configured through a drag-and drop interface. However, its flexibility comes at a cost, even when your import or export fails. For larger projects, such as company databases, where information needs to be consistantly moving, this is a great solution. However, for smaller projects, it can easily eat through the budget given for the Hackathon, despite it's effectiveness. As such, smaller projects are safer with building their own import/export functions with Lambda.
```

Comment:
```suggestion
As a turnkey solution, Data Pipeline works with most datasets, and can be configured through a drag-and drop interface. However, its flexibility comes at a cost, even when your import or export fails. For larger projects, such as company databases, where information needs to be consistantly moving, this is a great solution. However, for smaller projects, it can easily eat through the budget given for the Hackathon, despite its effectiveness. As such, smaller projects are safer with building their own import/export functions with Lambda.
```

Reply:
Addressed in most recent commit